react-game/src/components/sounds/soundTouch.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+
+
+const SoundTouch = ({musicTouch, playSoundTouch, plusVolume, minusVolume}) => {      
+       
+    const music = () => {
+        if (musicTouch) {
+            return (
+                <span className = "material-icons">
+                    music_note
+                </span>
+            )
+        } else {
+            return (
+                <span className = "material-icons">
+                    music_off
+                </span>
+            )
+        }
+    }
+
+    return (
+        <div className = 'd-flex'>
+            <button className= 'btn'
+                onClick={() => {playSoundTouch()}}>
+                {music()}
```

Comment:
It's a common practice to use ternary operator in React:
```suggestion
                <span className = "material-icons">
                      {musicTouch ? music_note : music_off}
                </span>
```